Libraries/libbom/Headers/bom/bom.h

```diff
@@ -39,7 +39,7 @@ bom_context_memory_file(const char *fn, bool writeable, size_t minimum_size);
 struct bom_context;
 
 struct bom_context *
-bom_alloc_empty(struct bom_context_memory memory);
+bom_alloc_size(struct bom_context_memory memory, uint32_t index_count);
```

Comment:
Honestly, I initially thought it was better to leave bom_alloc_empty() and add a new function so that we don't churn the existing API.

I'm going to go back to my initial instinct.